### PR TITLE
add Indonesian Locale

### PR DESCRIPTION
**Pull request by sdmoko**

### CODE REVIEW

File: pkg/colorful-indonesian.man [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.\" Contact veg@svgames.pl to correct errors or typos.
+.TH man 6 "2017-09-30" "1.2" "Manual Permainan"
+.SH NAMA
+colorful - Tembak-tembakan 2D sederhana
+.SH SINOPSIS
+\fBcolorful\fR
+.SH DESKRIPSI
+Colorful adalah permainan tembak-tembakan sederhana dari samping, dimana protagonis
+perjalanan labirin gua dan koridor untuk mendapatkan artifak warna.
+.SH Kontrol
```
Comment (suve) [inline pkg/colorful-indonesian.man]: This should be all uppercase.

File: pkg/colorful-indonesian.man [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Permainan ini dimainkan menggunakan papan tik. Tombol default:
+.TP
+\fBArrow keys\fR
```
Comment (suve) [inline pkg/colorful-indonesian.man]: Is this not translated, or do you actually use "arrow keys" in Indonesian?

File: pkg/colorful-indonesian.man [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Tembak kiri /kanan
+.TP
+\fB\-\fR/\fB+\fR
+Volume turun / naik
+.TP
+\fBP\fR
+Tunda / mulai
+.TP
+Tombol ini bisa diatur ulang seluruhnya.
+.SH DIRECTORIES
```
Comment (suve) [inline pkg/colorful-indonesian.man]: Same issue as with the "arrow keys".

File: pkg/colorful-indonesian.man [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Berkas permainan berlokasi di \fB/usr/share/suve/colorful\fR. 
+Permainan ini menyimpan pengaturan dan savegames di \fB~/.suve/colorful/\fR.
+.SH AUTHOR
```
Comment (suve) [inline pkg/colorful-indonesian.man]: Same issue as with "arrow keys" and "directories".

**Review by suve — CHANGES_REQUESTED** [review]
Apart from those minor issues, looks good!

**Comment by sdmoko** [comment]
@suve please check again, tell me if still wrong. :)
cheers

**Review by suve — APPROVED** [review]

**Comment by suve** [comment]
Looks fine. Merged. Thanks!

**Pull request merged**